### PR TITLE
Rating

### PR DESCRIPTION
**Pull request by AnjiManova**
- Rating page creation

### CODE REVIEW

File: en/components/rating.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+| Layer                        | Use                                                                                                                                                  |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 🚫 componentVersion | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it. |
+| 🚫 metadata         | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it. |
+| Label              |                                                                                                                 |
```
Comment (imincheva) [inline en/components/rating.md:23]: Missing info for the Label

File: en/components/rating.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+| Value              | Contains all the rating icons                                                                                                                 |
+
+After detaching, you may increase the number of the rating icons by duplicating any of the existing ones nested in the `Value` frame, and the size will be adjusted thanks to the Auto Layout.
+
+To decrease the number of the rating icons, there is no need to detach the component, you should hide the not needed one from the Layers panel.
+
+## Using the Rating in Sketch
+
+The Raiting component in Sketch has almost the same building elements like in Figma. There are Label and Value and the component comes with five rating icons by default. In case that there is need for more rating icons the component needs to be detached by right-clicking on the component and selecting `Detach from Symbol` from the contextual menu. After detaching you should see the following detached layers:
```
Comment (imincheva) [inline en/components/rating.md:32]: In case that there is need for more rating icons **,** the component needs...

File: en/components/rating.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+| Layer                        | Use                                                                                                                                                  |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 🚫 metadata        | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it.  |
+| Label              |                                                                                                                                                       |
```
Comment (imincheva) [inline en/components/rating.md:37]: Missing Label information

File: en/components/rating.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+The Rating component has an `Empty State` and it can be switched on and off from a boolean property in Figma. In addition, there is a `Value` property in Figma which is the opposite of the `Empty State` and allows you to select the fulfillment of the Rating Value from predefined options from 0 to 5 (supporting half star as well). In Sketch this can be achieved easily with Symbol Overrides.
+
+<img class="responsive-img" src="../images/rating_enabled_empty-state.png" srcset="../images/rating_enabled_empty-state@2x.png 2x" />
+<img class="responsive-img" src="../images/rating_value2.5.png" srcset="../images/rating_value2.5@2x.png 2x" />
+<img class="responsive-img" src="../images/rating_value5.png" srcset="../images/rating_value5@2x.png 2x" />
+
+## Size
+
+The Rating is available in tree sizes - Small, Medium and Large.
```
Comment (imincheva) [inline en/components/rating.md:64]: tree -> three

**Review by imincheva — CHANGES_REQUESTED** [review]

File: en/components/rating.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
+---
+title: Rating - Design System Component
+_description: The Rating component lets users apply a rating to an item or experience, such as an image, a forum post, an item for sale in a marketplace, and more.
+_keywords: Design Systems, Design Systems UX, UI kit, Figma, Figma to Angular, Export code from Figma, Figma to HTML, Figma UI kits, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
```
[the remark this answers is not in the document]
Comment (AnjiManova) [reply en/components/rating.md:4]: There are keywords for Figma.

File: en/components/rating.md (hunk 3 above)
Comment (AnjiManova) [reply en/components/rating.md:37]: In my opinion, there is no need to add a description for the label, but I could add it for more clarity

File: en/components/rating.md (hunk 1 above)
Comment (AnjiManova) [reply en/components/rating.md:23]: In my opinion, there is no need to add a description for the label, but I could add it for more clarity

File: en/components/rating.md (hunk 2 above)
Comment (AnjiManova) [reply en/components/rating.md]: Fixed in the next commit

File: en/components/rating.md (hunk 1 above)
Comment (AnjiManova) [reply en/components/rating.md:23]: Added information for the label in the next commit

**Review by imincheva — DISMISSED** [review]

File: en/components/rating.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+| 🚫 metadata        | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it.  |
+| Label              | Text layer that can be changed or hidden if not needed                                                                                                                                                      |
+| Value              | Contains all the rating icons                                                                                                                         |
+
+In order to increase or decrese the number of rating icons, you should use the `Detach from Sybmol` option again over the `Value` symbol and then by duplicating or deleting any of the rating icons that already exist you will be able to achive the desired amount of icons.
+
+## Using the Rating in Adobe XD
+
+> [!NOTE]
+> WIP: Currently, we don't support the Rating component in Adobe XD.
```
Comment (IGSvt) [inline en/components/rating.md:45]: Adobe XD -> Adobe Xd

File: en/components/rating.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+## Interaction States
+
+The Rating can be inserted in an enabled or disabled state.
```
Comment (IGSvt) [inline en/components/rating.md:49]: In Sketch the Rating can be inserted in an enabled or disabled state, while in Figma when inserted the Rating component is always enabled by default, and in order to disable it you can do that from the property panel.

File: en/components/rating.md (hunk 6 above)
Comment (AnjiManova) [reply en/components/rating.md:45]: The official name is with two capital letters

File: en/components/rating.md (hunk 7 above)
Comment (AnjiManova) [reply en/components/rating.md:49]: Fixed

Comment (imincheva) [reply en/components/rating.md]: @AnjiManova it should actually be properties panel, not property, Vasil misled you 😃

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**